README.md

```diff
@@ -364,22 +444,50 @@ It is often best to default to structs, only falling back to classes when you sp
 
 ```swift
 func setUpUI() {
-    view.backgroundColor = UIColor.blue()
+
+    view.backgroundColor = .blue
 }
 ```
 
 *Not Preferred:*
 
 ```swift
 func setUpUI() {
-    self.view.backgroundColor = UIColor.blue()
+
+    self.view.backgroundColor = .blue
 }
 ```
 
 *Rationale: Omitting `self` allows for more concise code.  Only use it when a local variable has hidden an instance variable or in escaping closures.*
 
 ---
 
+##### Use dot syntax on enums and static vars where type can be inferred.
```

Comment:
Is "static vars" the right phrase for this?  Should these be "static properties"?  And is dropping the type specification referred to as "dot syntax"?  I'm curious if there's a more accurate name for this shortening.